nucypher/blockchain/eth/sol/source/contracts/proxy/Upgradeable.sol

```diff
@@ -32,6 +32,11 @@ abstract contract Upgradeable is Ownable {
     */
     uint8 public isUpgrade;
 
+    /**
+    * @dev Guarantees that next slot will be separated from the previous
+    */
+    uint256 stubSlot;
```

Comment:
It could only increase gas for handling previous slot. We don't need to put everything in fewer slots, Ideally we need to update at most one slot per one tx. Also this `stubSlot` is free